src/CodeCompliance/Domain/Checks/Filters/PluginFilter.php

```diff
@@ -17,6 +17,16 @@ class PluginFilter implements FilterInterface
      */
     public const PLUGIN_FILTER = 'PLUGIN_FILTER';
 
+    /**
+     * @var string
+     */
+    protected const PLUGIN_CLASS_SUFFIX = '/.*Plugin$/';
```

Comment:
You can use .+ since it cannot be ever empty string before

Reply:
++ 
Thanks a lot